Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -746,6 +746,16 @@ def decode_type(type):
         if len(decodable_classes) == 1:
             match = re.search("RetainPtr<(.*)>", member.type)
             assert match
+            for attribute in member.attributes:
+                precondition = re.search(r'Precondition=\'(.*)\'', attribute)
+                if precondition:
```

Comment:
It looks like WebKit tooling is at Python 3.9.6 right now? If this is not just my desk config, I think we can adopt the walrus operator here. It's the Python parallel to C++ init statements. 🙂

```suggestion
                if precondition := re.search(r'Precondition=\'(.*)\'', attribute):
```